lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -23,6 +24,10 @@ def string_to_array(string)
         return string_array
       end
 
+      def parse_plist(path)
+        CFPropertyList.native_types(CFPropertyList::List.new(file: path).value)
+      end
+
       def get_ios_app_id_from_archive(path)
```

Comment:
Do we have a test for this? It would be great if this bug was caught by a failing test 🙂